### DISCUSSION TITLE
`SectionList` with `numOfCols`

### DISCUSSION THREAD

**Discussion by Stringsaeed** [Shopify/flash-list#554] [Q&A] [answered]
How to achieve `SectionList` with `numbOfCol`

**Comment by naqvitalha** [comment, marked as answer]
You can use `overrideItemLayout` prop to set a higher column span value for your sticky sections. For example, if you have 2 columns, span of sticky headers should be 2 to give them the right look.

**Reply by Stringsaeed** [reply]
@naqvitalha can you elaborate with an example, please?

**Reply by Stringsaeed** [reply]
@naqvitalha I tried it but it had a weird RTL issue, I will report it
